com.jetbrains.WebStorm.yaml

```diff
@@ -27,35 +27,37 @@ modules:
   - name: WebStorm
     buildsystem: simple
     build-commands:
+      - unzip webstorm_logos.zip
       - install -D apply_extra /app/bin/apply_extra
-      - install -Dm755 webstorm-desktop "${FLATPAK_DEST}/bin/webstorm-desktop"
-      - install -Dm644 icon.svg "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg"
-      - icon_in=icon.svg;
+      - install -Dm755 webstorm-desktop.sh "${FLATPAK_DEST}/bin/webstorm-desktop"
+      - install -Dm644 logo.svg "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg"
+      - icon_in=logo.svg;
```

Comment:
Is this (until the `done` line) really needed? I thought that providing a (real) `.svg` file was key for the Flatpak builder to correctly convert and place the icons with the required sizes.